lib/ansible/playbook/helpers.py

```diff
@@ -112,10 +112,11 @@ def load_list_of_tasks(ds, play, block=None, role=None, task_include=None, use_h
                 # 1. the user set the 'static' option to true
                 # 2. one of the appropriate config options was set
                 # 3. the included file name contains no variables, and has no loop
-                is_static = t.static or \
-                            C.DEFAULT_TASK_INCLUDES_STATIC or \
-                            (use_handlers and C.DEFAULT_HANDLER_INCLUDES_STATIC) or \
-                            not templar._contains_vars(t.args.get('_raw_params')) and t.loop is None
+                if t.static is not None:
+                    is_static = t.static
```

Comment:
Ahh ok, fair enough :) I knew we still had the loop check below as well, forgot we errored on templating there too.